scripts/build-binary

```diff
@@ -24,7 +24,7 @@ VERSION_FLAGS="
         -X ${K3S_PKG}/pkg/version.Program=${PROG}
         -X ${K3S_PKG}/pkg/version.Version=${VERSION}
         -X ${RKE2_PKG}/pkg/images.DefaultRegistry=${REGISTRY}
-        -X ${RKE2_PKG}/pkg/images.DefaultEtcdImage=rancher/hardened-etcd:${ETCD_VERSION}-${IMAGE_BUILD_VERSION}
+        -X ${RKE2_PKG}/pkg/images.DefaultEtcdImage=rancher/hardened-etcd:${ETCD_VERSION}-build20211118
```

Comment:
We should retain the usage of `${IMAGE_BUILD_VERSION}` should we not?

Reply:
I changed it because I saw in master was like this https://github.com/rancher/rke2/blob/2faa95b6a8b3a3abe12fb90c4529f7aba29803da/scripts/build-binary#L27

Reply:
It made sense to have a variable when it looked like we were going to build all the images at the same time, but now with things being as ad-hoc as they are - many images have different build tags -  it's not really getting used as I thought it would. I'm OK with dropping it.